modules/prebidServerBidAdapter/index.js

```diff
@@ -558,10 +571,27 @@ const OPEN_RTB_PROTOCOL = {
             bidRequest.serverResponseTimeMs = serverResponseTimeMs;
           }
 
+          // if ext.prebid.targeting exists append adserverTargeting
+          if (utils.deepAccess(bid, 'ext.prebid.targeting')) {
+            bidObject.adserverTargeting = bid.ext.prebid.targeting;
+          }
+
           if (utils.deepAccess(bid, 'ext.prebid.type') === VIDEO) {
             bidObject.mediaType = VIDEO;
+
+            // try to get cache values from 'response.ext.prebid.cache'
+            // else try 'bid.ext.prebid.targeting' as fallback
+            if (bid.ext.prebid.cache && typeof bid.ext.prebid.cache.vastXml === 'object' && bid.ext.prebid.cache.vastXml.cacheId && bid.ext.prebid.cache.vastXml.url) {
+              bidObject.videoCacheKey = bid.ext.prebid.cache.vastXml.cacheId;
+              bidObject.vastUrl = bid.ext.prebid.cache.vastXml.url;
+            } else if (typeof bid.ext.prebid.targeting === 'object' && bid.ext.prebid.targeting.hb_uuid && bid.ext.prebid.targeting.hb_cache_hostpath) {
```

Comment:
Here is an example of the targeting values PBS passes back:

![image](https://user-images.githubusercontent.com/13972794/52425059-15364680-2ab0-11e9-899f-4e75987513d6.png)

So we need to get an update on if `uuid` in fact will be removed for cache_key because as of now it is only returned on banner cached bids i think...